cpp/account_server.cc

```diff
@@ -75,6 +76,15 @@ class AccountServiceImpl final : public Account::Service {
   std::string hostname_;
 };
 
+std::unique_ptr<Server> StartAdminServer(const std::string& port) {
```

Comment:
Shouldn't we remove this from all languages if it's not in Go?

Reply:
In the observability guide, I want the example to just work. There's no change needed for the deployed cluster. So, the admin interface will be turned on by default for C++/Java/Go.

According to you, Golang's admin interface doesn't work because https://cloud.google.com/traffic-director/docs/proxyless-configure-advanced-traffic-management#proxyless-traffic-example-templatel failed to include xds_bootstrap file generation. But it only apply to `account-service`.

So, admin interface is still available in `stats-service` and `wallet-service`. The observability guide uses a VM that runs `wallet-service` to explain how `grpcdebug` works.

Reply:
(Boostrap file generation is included `create_service.sh`.)
https://github.com/GoogleCloudPlatform/traffic-director-grpc-examples/blob/664eeddd149afce7eb8945f758380a7d0657dd5e/scripts/create_service.sh#L51

Reply:
Sorry, I misunderstood the question. You didn't mean removing admin from all servers cross all languages. You mean removing admin from all account servers.

Admin interface works without xds_bootstrap file in C++/Java. I think it's fine to keep them? What's the benefit of removing them?

Reply:
Done.